components/admin_console/remove_file_setting.jsx

```diff
@@ -22,12 +21,18 @@ export default class RemoveFileSetting extends Setting {
         };
     }
 
+    constructor(props) {
+        super(props);
+
+        this.remove_button = React.createRef();
```

Comment:
Two things regarding naming here:
1.  We prefer to use lower camel case for variables names
2. Typically we append 'Ref' to the end of these variable names

Could we please get the name of this variable changed to something like `removeButtonRef`?

```suggestion
        this.removeButtonRef = React.createRef();
```